src/cheri/debug-integration.adoc

```diff
@@ -228,9 +242,8 @@ xref:default-csrnames-added[xrefstyle=short].
 .Debug default data capability
 include::img/dddcreg.edn[]
 
-Upon entry to debug mode, <<ddc>> is saved in <<dddc>>. <<ddc>>'s metadata is
-set to the <<infinite-cap>> capability's metadata (with tag set) and <<ddc>>'s
-address remains unchanged.
+Upon entry to debug mode, <<ddc>> is saved in <<dddc>>. <<ddc>> is set to a
+<<root-rw-cap>> capability with its address set to that of the just-updated <<dddc>>.
```

Comment:
I quite like the previous wording of "address remains unchanged" - is there anything wrong with that?

Reply:
It's a little unusual to my mind to talk about "setting the capability metadata", in that it feels like violating abstraction boundaries.  But no, it's fine and I've put it back (I think acceptably).